core/src/main/resources/hudson/triggers/TimerTrigger/help-spec.jelly

```diff
@@ -71,15 +71,15 @@
     Examples:
   </p>
 <pre>
-# every fifteen minutes (perhaps at :07, :22, :37, :52)
+# Every fifteen minutes (perhaps at :07, :22, :37, :52):
 H/15 * * * *
-# every ten minutes in the first half of every hour (three times, perhaps at :04, :14, :24)
+# Every ten minutes in the first half of every hour (three times, perhaps at :04, :14, :24):
 H(0-29)/10 * * * *
-# once every two hours at 45 minutes past the hour starting at 9:45 AM and finishing at 3:45 PM every weekday.
+# Once every two hours at 45 minutes past the hour starting at 9:45 AM and finishing at 3:45 PM every weekday:
 45 9-16/2 * * 1-5
-# once in every two hours slot between 9 AM and 5 PM every weekday (perhaps at 10:38 AM, 12:38 PM, 2:38 PM, 4:38 PM)
-H H(9-16)/2 * * 1-5
-# once a day on the 1st and 15th of every month except December
+# Once in every two hour slot between 8 AM and 4 PM every weekday (perhaps at 9:38 AM, 11:38 AM, 1:38 PM, 3:38 PM):
+H H(8-15)/2 * * 1-5
```

Comment:
Are you sure this can ever be odd hours? Have you tried this?

Reply:
I haven't tried. But if it doesn't support this, then the description should be change to explain that you can't do this.

Normal crontab apparently doesn't distinguish between odd/even:
[“At minute 0 past every 2nd hour from 8 through 15.”](https://crontab.guru/#0_8-15/2_*_*_*)
[“At minute 0 past every 2nd hour from 9 through 16.”](https://crontab.guru/#0_9-16/2_*_*_*)

And note that one of the items talked about the problem of using `days/3`.

Reply:
I don't get why the description says from 8 to 4 and the expression is from 8 to 3. The links you provided have a congruent description and expression, not like our code. I mean, the comment was wrong before and now it stays so. WDYT?

Reply:
I tried to reason through the comment and gave up. I had a theory but... I probably should have opened a ticket about it.

Reply:
@MRamonLeon See the examples provided (which are a possibility depending on how it's implemented, I'm just not confident it works as explained). Clearly 3:38 PM isn't before 3 PM.

Reply:
@oleg-nenashev What specifically is the defect?

Reply:
Sorry, I was referring to 

> Normal crontab apparently doesn't distinguish between odd/even:

IIRC we had issues with that in the past. Anyway, I do not block it in any means